### DISCUSSION TITLE
如何设置边的类型为贝塞尔？

### DISCUSSION THREAD

**Discussion by kinop112365362** [didi/LogicFlow#1336] [Q&A] [answered]
现在默认拖出的都是直线，如何设置成默认贝塞尔？

**Comment by wumail** [comment, marked as answer]
```typescript
new LogicFlow({
 ...
 edgeType: 'biezer',
 ...
})

// or 

lf.setDefaultEdgeType('bezier')
``` 
参考文档：
https://site.logic-flow.cn/docs/#/zh/guide/basic/edge?id=%e8%87%aa%e5%ae%9a%e4%b9%89%e8%8a%82%e7%82%b9%e4%b9%8b%e9%97%b4%e8%be%b9%e7%9a%84%e7%b1%bb%e5%9e%8b
https://site.logic-flow.cn/docs/#/zh/api/logicFlowApi?id=setdefaultedgetype
